live-build/misc/ansible-roles/appliance-build.delphix-autofs/files/net.autofs

```diff
@@ -0,0 +1 @@
+/net	/etc/auto.net
```

Comment:
A general convention in ansible roles is to structure the "files" directory similar to where the files will be placed on the root filesystem. E.g. in this example, we'd have:
```
appliance-build.delphix-autofs/files/etc/autofs.master.d/net.autofs
```
then we can do this in the `main.yml` file to copy all files in that "files" directory:
```
- copy:                                                                         
    src: etc/                                                                   
    dest: /etc/                                                                 
    owner: root                                                                 
    group: root                                                                 
    mode: 0444  
```

with that said, it doesn't _really_ matter (this works), and we're already breaking that convention in at least one other place in this repository, but I just wanted to mention it.

Reply:
Yes, both of these suggestions make sense to me, I'll update shortly.